_glaredb/hybrid-execution.md

```diff
@@ -0,0 +1,132 @@
+---
+layout: default
+title: Hybrid Execution
+nav_order: 3
+---
+
+# Hybrid execution
+
+Connecting to a Cloud deployment through the [Python Library] or [CLI] will
+enable hybrid execution of SQL queries. This enables a query to utilize the
+resources of both your local machine and the Cloud deployment during execution.
+This also allows querying data across your Cloud deployment and your local
+file system.
+
+Let's take a look at the following query:
+
+```sql
+select m.user_id,
+  max(m.output_rows),
+  avg(m.output_rows)::int
+from execution_metrics m
+  inner join './company_users.csv' u on m.user_id = u.id
+group by m.user_id
+limit 5;
+```
+
+In this example, `execution_metrics` is table that exists on the Cloud
+deployment, and `./company_users.csv` is a CSV file that's sitting on your
+laptop. With hybrid execution, the query gets executed across both your local
+machine and Cloud deployment while minimizing the amount of data transferred
+across machines.
+
+## Getting started with hybrid execution
+
+Hybrid execution requires a [Deployment] on GlareDB Cloud. Creating a deployment
+is quick and easy.
+
+Once you have a deployment ready to go, open the "Connect" dialog to get a set
+of credentials for connecting to your deployment. This dialog provides the
+commands needed to connect to your deployment using either the CLI or Python
+library.
+
+![Connect dialog](/assets/images/connect-dialog.png)
+
+Connecting via the CLI can be done with the `\open` command:
+
+```text
+> \open glaredb://user:password@hello.remote.glaredb.com:6443/hello
+Connected to Cloud deployment: hello
+```
+
+And connecting via the Python library can be done with the `connect` method:
+
+```python
+import glaredb
+con = glaredb.connect("glaredb://user:password@hello.remote.glaredb.com:6443/hello")
+```
+
+Once connected, you're able to query tables on your Cloud deployment, connected
+data sources like [Postgres] and [Snowflake] databases, objects in S3, and files
+on your local machine. All this without having to move data around.
```

Comment:
I think we can hold off until we've fleshed that out a bit, since I think that would be a great update for our homepage, and the GlareDB repo as well. I also think it would come with some devrel/tutorial/examples.

It's a good suggestion, we can add it to our internal roadmap